tests/integration-tests.rs

```diff
@@ -144,11 +49,11 @@ async fn insert_identity_and_proofs() {
         .await
         .expect("Failed to spawn ganache chain");
 
-    options.app.ethereum.ethereum_provider =
+    options.app.ethereum.read_options.ethereum_provider =
         Url::parse(&chain.endpoint()).expect("Failed to parse ganache endpoint");
     options.app.contracts.semaphore_address = semaphore_address;
-    options.app.ethereum.signing_key = private_key;
-    options.app.ethereum.confirmation_blocks_delay = 2;
+    options.app.ethereum.write_options.signing_key = private_key;
```

Comment:
Probably need to fix clippy config? Looks like it's broken by the flags.

Reply:
Fixed in https://github.com/worldcoin/signup-sequencer/pull/367 (feel free to review, too!)